src/gui/base/BubbleTextField.js

```diff
@@ -150,21 +158,33 @@ export class BubbleTextField<T> {
 					}
 				}),
 				m(`.suggestions.abs.z4.full-width.elevated-bg.scroll.text-ellipsis${this.suggestions.length ? ".dropdown-shadow" : ""}`, {
-					oncreate: vnode => this._domSuggestions = vnode.dom,
+					oncreate: vnode => {
+						this._domSuggestions = vnode.dom
+						this._selectedSuggestionChangedListener = this.selectedSuggestion.map(
+							makeListSelectionChangedScrollHandler(this._domSuggestions, this.bubbleHandler.suggestionHeight, this._getSelectedSuggestionIndex.bind(this))
+						)
+					},
+					onbeforeremove: () => {
+						this._selectedSuggestionChangedListener.end()
```

Comment:
this should be `this._selectedSuggestionChangedListener.end(true)`

Reply:
Sure. I will change it.